src/OpenTracing/Basic/Span.php

```diff
@@ -0,0 +1,175 @@
+<?php
+
+namespace OpenTracing\Basic;
+
+use OpenTracing;
+
+
+class Span extends OpenTracing\Span
+{
+
+    private $data = null;
+
+    public function __construct(OpenTracing\Tracer $tracer, SpanData $data)
+    {
+        $this->tracer = $tracer;
+        $this->data = $data;
+    }
+
+    /**
+     * Sets or changes the operation name.
+     *
+     * @param string $operationName
+     * @return $this
+     */
+    public function setOperationName($operationName)
+    {
+        $this->data->operationName = $operationName;
+
+        return $this;
+    }
+
+    /**
+     * Indicates that the work represented by this span has been completed
+     * or terminated, and is ready to be sent to the Reporter.
+     *
+     * If any tags / logs need to be added to the span, it should be done
+     * before calling finish(), otherwise they may be ignored.
+     *
+     * @param float $finishTime
```

Comment:
please clarify the precision of this value in the comments (e.g. seconds since Unit epoch or whichever the convention is), and reference the sys function used to obtain it.